src/presentation/api/AdminController.test.ts

```diff
@@ -0,0 +1,52 @@
+import request from 'supertest';
+import { cleanupDatabase } from '../../test/cleanupDatabase';
+import database from '../../database';
+import { getTokenForUser } from '../../test/authentication';
+import { ADMIN, USER } from '../../domain/model/authentication/Roles';
+import { BULK_CREATE_USERS } from '../../domain/model/authentication/Permissions';
+import { persistedUserWithRoleAndPermissions } from '../../test/persistedEntities';
+import { Application } from '../../presentation/Application';
+
+describe('admin endpoints', () => {
+  const app = new Application().getExpressApp();
+
+  beforeEach(async () => {
+    await cleanupDatabase();
+  });
+
+  describe('POST /users', () => {
+    it('successfully creates new users', async () => {
+      const authenticatedUser = await persistedUserWithRoleAndPermissions(ADMIN, [BULK_CREATE_USERS]);
+
+      await request(app)
+        .post(`/api/v1/admin/users`)
```

Comment:
shouldn't this be `bulk-users` ?

Reply:
Hmm.. Ignore me.. You prefixed it with `/admin`. Makes sense 👍

Reply:
As it's inside of the /admin prefix, I think a POST to /users is enough (as the only reason to create users behing the admin endpoint is to do it bulk, even if you only create one). But I don't have strong preferences... `bulk-users` doesn't sound very RESTfully, that's why I didn't use it in a first instance

Reply:
Under `/admin`, I prefer just `users` as well 👍